portia-entrypoint

```diff
@@ -175,17 +175,16 @@ def main():
     env = update_environment_for_portia()
     _print_flush("exec env: %s" % env)
     # sys.argv[0] contains 'portia-entrypoint' as it's called directly
-    cmd = os.path.basename(sys.argv[1])
-    if cmd == 'scrapy-list':
-        os.execvpe('scrapy', ['scrapy', 'list'], env)
-    elif cmd == 'start-crawl':
-        # Call sh_scrapy's start-crawl
-        # Replacing the process is needed so python obeys PYTHONPATH,
-        # The alternative is to modify sys.path directly but we rely
-        # on PYTHONPATH already for "scrapy list".
-        os.execve('/usr/local/bin/start-crawl', [cmd], env)
-    elif len(sys.argv) > 2:
-        os.execve(sys.argv[1], sys.argv[1:], env)
+    if len(sys.argv) > 1:
+        cmd = os.path.basename(sys.argv[1])
+        if cmd == 'start-crawl':
```

Comment:
Please, note, that I dropped logic for `scrapy-list` here, looks like it's useless as for now we don't use portia-stack with builder.